bin/stacks/api-stack.ts

```diff
@@ -367,6 +370,13 @@ export class APIStack extends cdk.Stack {
       },
     })
 
+    const limitOrders = api.root.addResource('limit-orders', {
```

Comment:
```suggestion
    const limitOrders = api.root.addResource('limit', {
```
nit: Feel like `/limit` might be better because we also add on `order` already for the next resource?

Reply:
also maybe matches a little better with `dutch-auction/order`